app/react/kubernetes/ingresses/CreateIngressView/IngressForm.tsx

```diff
@@ -367,13 +357,23 @@ export function IngressForm({
                             handleTLSChange(hostIndex, e.target.value)
                           }
                           defaultValue={host.Secret}
+                          className="!rounded-r-none"
                         />
+                        {!host.NoHost && (
+                          <div className="input-group-btn">
+                            <Button
+                              className="btn btn-light btn-sm !ml-0 !rounded-l-none"
```

Comment:
The Button api has a few props that overlap.
'btn' is applied by default
size="small" (the default size) will apply 'btn-sm'
color="default" will apply 'btn-light'